packages/uswds-core/src/styles/settings/_settings-general.scss

```
@@ -112,3 +112,11 @@ Icons
 */
 
 $theme-icon-image-size: 2 !default;
+
+/*
+----------------------------------------
+High Contrast
```

Wondering if it would be helpful to specify that this is for High contrast _mode_, possibly with a short description explaining HC mode/CSS forced-color mode is and how to activate it. 
```suggestion
High contrast mode
```

---

Resolved in [b2f35e6](https://github.com/uswds/uswds/pull/5147/commits/b2f35e6583ca2674cd9299897d0b5e4cada5bdbc)